src/Article.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import './Article.css';
+import featuredimage from './image-3-medium.jpg';
+
+class Article extends Component {
+  render() {
+  	const toggledClass = this.props.expanded ? 'expanded' : 'collapsed';
+    var image = require(`${this.props.article.featured_image}`);
+    return (
+      <div className="c-mobile-article">
```

Comment:
Use single quotes everywhere

Reply:
Do you mean for the class attribute? In the ubyssey.ca repo, the class attribute uses double quotation marks so I'm just following that.